Enzyme break after React upgrade from 15.0.0-rc2 to 15.0.1

We are using enzyme to test our React components together with browserify.
But after upgrade React from rc2 to stable 15.0.1:

If i'll run tests, i'll get this error:
`Error: Cannot find module 'react/addons' from '/myproject/node_modules/enzyme/build`

But installed version is 15.0.1 (React.version) which is using module `react-addons-test-utils`
If i'll remove this line:
`TestUtils = require('react/addons').addons.TestUtils;`
from `build\react-compat.js` its works properly.

Package.json:
```
"dependencies": {
    "react": "15.0.1",
    "react-addons-perf": "15.0.1",
    "react-addons-test-utils": "15.0.1",
    "react-addons-update": "15.0.1",
    "react-dom": "15.0.1"
  },
  "devDependencies": {
    "babel-core": "^6.7.0",
    "babel-plugin-external-helpers-2": "^6.3.13",
    "babel-plugin-transform-decorators-legacy": "^1.3.4",
    "babel-polyfill": "^6.3.14",
    "babel-preset-es2015": "^6.6.0",
    "babel-preset-react": "^6.5.0",
    "babel-preset-stage-0": "^6.5.0",
    "babelify": "^7.2.0",
    "browserify": "^13.0.0",
    "enzyme": "^2.2.0",
    "watchify": "^3.7.0"
  }
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -22,7 +22,7 @@ npm install && npm run react:14
 ```
 
 
-### Switching between React 0.14 and React 0.13
+### Switching between React 15, React 0.14 and React 0.13
 
 ```bash
 # switch to React 0.13
@@ -32,6 +32,11 @@ npm run react:13
 ```bash
 # switch to React 0.14
 npm run react:14
+```
+
+```bash
+# switch to React 15
+npm run react:15
 ```
 
 
@@ -86,8 +91,8 @@ Before you submit a pull request from your forked repo, check that it meets thes
 1. If the pull request fixes a bug, it should include tests that fail without the changes, and pass
 with them.
 1. If the pull request adds functionality, the docs should be updated as part of the same PR.
-1. The pull request should work for both React 0.14 and React 0.13. The CI server should run the
-tests in both versions automatically when you push the PR, but if you'd like to check locally, you
+1. The pull request should work for React 15, React 0.14 and React 0.13. The CI server should run the
+tests in all versions automatically when you push the PR, but if you'd like to check locally, you
 can do so (see above).
 1. Please rebase and resolve all conflicts before submitting.
 

--- a/docs/guides/browserify.md
+++ b/docs/guides/browserify.md
@@ -14,6 +14,7 @@ In your browserify configuration, you simply need to make sure that the followin
 labeled as "external", which means they will be ignored:
 
 ```
+react/addons
 react/lib/ReactContext
 react/lib/ExecutionEnvironment
 ```
@@ -26,6 +27,7 @@ var browserify = require('browserify');
 var b = browserify();
 
 // make sure to mark these as external!
+b.external('react/addons');
 b.external('react/lib/ReactContext');
 b.external('react/lib/ExecutionEnvironment');
 

--- a/docs/guides/karma.md
+++ b/docs/guides/karma.md
@@ -30,6 +30,7 @@ webpack: { //kind of a copy of your webpack config
   },
   externals: {
     'cheerio': 'window',
+    'react/addons': true,
     'react/lib/ExecutionEnvironment': true,
     'react/lib/ReactContext': true
   }
@@ -49,6 +50,7 @@ browserify: {
   ],
   configure: function(bundle) {
     bundle.on('prebundle', function() {
+      bundle.external('react/addons');
       bundle.external('react/lib/ReactContext');
       bundle.external('react/lib/ExecutionEnvironment');
     });

